### PR TITLE
Draft: Added VP calculation. Added income level logic. Updated actions

### PR DESCRIPTION
**Pull request by tylermcdonald**

### CODE REVIEW

File: app/environments/brassbirmingham/brassbirmingham/envs/test.py [hunk 1]
```diff
@@ -143,10 +138,134 @@ def testNetworkSearch(self):
...
+        # Big price jump
+        board.coalMarketRemaining = 13
+        assert board.priceForCoal(10) == 35
+        board.ironMarketRemaining = 8
+        assert board.priceForIron(10) == 40
+
+    def testVictoryPoints(self):
+        # Zero points
+        resetGame(2)
+        playerVPs = board.getVictoryPoints()
```
Comment (noahpodgurski) [inline app/environments/brassbirmingham/brassbirmingham/envs/test.py]: Wouldn't it make more sense to just attach a `vp` attr to the `Player` object?

File: app/environments/brassbirmingham/brassbirmingham/envs/classes/board.py [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
 
 from consts import (
     CANAL_PRICE,
+    MAX_MARKET_COAL,
+    MAX_MARKET_IRON,
     ONE_RAILROAD_COAL_PRICE,
     ONE_RAILROAD_PRICE,
     ROAD_LOCATIONS,
     STARTING_CARDS,
-    STARTING_WILD_BUILDING_CARDS,
```
Comment (noahpodgurski) [inline app/environments/brassbirmingham/brassbirmingham/envs/classes/board.py:11]: Are we **absolutely** sure there's no limit to these wild cards?

Comment (tylermcdonald) [reply app/environments/brassbirmingham/brassbirmingham/envs/classes/board.py:11]: Yep. The number of each wild card is equal to the number of players playing and no one can take more than 1 of each at once

File: app/environments/brassbirmingham/brassbirmingham/envs/test.py (hunk 1 above)
Comment (tylermcdonald) [reply app/environments/brassbirmingham/brassbirmingham/envs/test.py]: VPs are only incremented at the end of each era and it's effectively a "board-based" calculation, in my opinion which is why I added it to board.

Player objects will store their intermediate VPs from Canal to Rail era though

**Pull request merged**